### PR TITLE
Allow Origin * by default

### PR DESCRIPTION
**Pull request by sven-urbanski-freiheit-com**
This is relevant for the local setup.
When using the helm chart, the value is automatically set

### CODE REVIEW

File: services/frontend-service/pkg/config/config.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ type ServerConfig struct {
 	AzureRedirectUrl    string `default:"" split_words:"true"`
 	Version             string `default:""`
 	SourceRepoUrl       string `default:"" split_words:"true"`
-	AllowedOrigins      string `default:"" split_words:"true"`
+	AllowedOrigins      string `default:"*" split_words:"true"`
```
[the remark this answers is not in the document]
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/pkg/config/config.go]: done

[the remark this answers is not in the document]
Comment (hannesg) [reply services/frontend-service/pkg/config/config.go]: I'm 99% sure this doesn't work. You have to give here either a fixed origin like "http://localhost:3000" or "*" or "null".

File: services/frontend-service/pkg/config/config.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ type ServerConfig struct {
 	AzureRedirectUrl    string `default:"" split_words:"true"`
 	Version             string `default:""`
 	SourceRepoUrl       string `default:"" split_words:"true"`
-	AllowedOrigins      string `default:"" split_words:"true"`
+	AllowedOrigins      string `default:"localhost:*" split_words:"true"`
```
Comment (hannesg) [inline services/frontend-service/pkg/config/config.go:34]: Is that acutally the right place for this default? We only need it on localhost, so maybe rather put it in the makefile?

File: services/frontend-service/pkg/config/config.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/pkg/config/config.go]: I tried "localhost:*" it works too

File: services/frontend-service/pkg/config/config.go (hunk 2 above)
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/pkg/config/config.go:34]: yes, you are right. I am moving it to the makefile

**Review by hannesg — APPROVED** [review]

**Pull request merged**